app/assets/stylesheets/components/_markdown-editor.scss

```diff
@@ -27,7 +27,7 @@ $app-toolbar-button-size: 50px;
 .app-c-markdown-editor__preview-toggle {
   width: 100%;
   border-bottom: $govuk-border-width-form-element solid $govuk-border-colour;
-  background-color: govuk-colour("grey-3");
```

Comment:
I'm assuming `grey-3` is now equivalent to `grey-4`.

Reply:
Yes, now both `grey-3` and `grey-4` are replaced by `light-grey` which makes them the same.